loleaflet/admin/admin.strings.js

```diff
@@ -26,6 +26,7 @@ l10nstrings.strViewers = _('Views');
 l10nstrings.strElapsedTime = _('Elapsed time');
 l10nstrings.strIdleTime = _('Idle time');
 l10nstrings.strModified = _('Modified');
+l10nstrings.strWopiServer = _('WOPI Server');
```

Comment:
I know that WOPI Server = WOPI host, but can we use the same term (WOPI host) everywhere, please?

Reply:
Done : )